### ISSUE TITLE
Can the image run as proxy?

### ISSUE THREAD

**Issue by alantang888** [zabbix/zabbix-community-docker#36]
Because run Zabbix on distributed environment usually will use proxy, but don't know can this image run as proxy, or will have another image for proxy.

**Comment by jangaraj** [comment]
Proxy is not available at the moment. However image is designed for proxy as well and there is idea to proxy in the future. You should to use another solution for now.

**Comment by jangaraj** [comment]
Initial proxy support (only mysql proxy DB) has been implemented - for now is available only in dev image:

```
docker run \
    -d \
    --name zabbix-db \
    --env="MARIADB_USER=zabbix" \
    --env="MARIADB_PASS=my_password" \
    zabbix/zabbix-db-mariadb
docker run \
    -d \
    --name zabbix-proxy \
    -p 10051:10051 \
    -v /etc/localtime:/etc/localtime:ro \
    --link zabbix-db:zabbix.db \
    --env="ZP_DBHost=zabbix.db" \
    --env="ZP_DBUser=zabbix" \
    --env="ZP_DBName=zabbix" \
    --env="ZP_DBPassword=my_password" \
    --env="ZP_DBPort=3306" \
    --env="ZP_Server=127.0.0.1" \
    --env="ZP_AllowRoot=1" \
    --env="ZP_enabled=true" \
    --env="ZA_enabled=false" \
    --env="ZW_enabled=false" \
    --env="ZS_enabled=false" \
    zabbix/zabbix-3.0:dev
```

Feel free to test.

**Comment by jangaraj** [comment]
Zabbix proxy (mysql only) is available in all zabbix-3.0 Docker images. Documented in README. Example available also in Zabbix wiki http://zabbix.org/wiki/Dockerized_Zabbix#Dockerized_Zabbix_Proxy_3.0
